src/api/db/models/Batch.ts

```diff
@@ -56,7 +56,7 @@ export class BatchModel {
   }
 
   static async queryById(_id: string): Promise<mongoose.HydratedDocument<BatchSchemaWithErrors>> {
-    const query = { _id };
+    const query = { _id: { $eq: _id } };
```

Comment:
@alukach This makes sense to me, but just posing the question is there anything wrong with how this is done?

Reply:
Honestly, it's not obvious that anything was wrong with the previous technique, buy CodeQL complained on another PR: https://github.com/tnc-ca-geo/animl-api/pull/220#discussion_r1642957203